eslint.config.mjs

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+import eslint from "@eslint/js";
+import tseslint from "typescript-eslint";
+import pluginReact from "eslint-plugin-react";
+import eslintPluginPrettierRecommended from "eslint-plugin-prettier/recommended";
+
+export default [
```

Comment:
Could you use `tseslint.config` here, like we are in https://github.com/alveusgg/alveusgg/blob/main/apps/website/eslint.config.js please (it will give us proper type-checking).